src/pages/Account/Tabs/ModulesTab/Contract.tsx

```diff
@@ -296,7 +297,12 @@ function RunContractForm({
       type: "entry_function_payload",
       function: `${module.address}::${module.name}::${fn.name}`,
       type_arguments: data.typeArgs,
-      arguments: data.args,
+      arguments: data.args.map((arg, i) => {
+        // use i+1 because the first argument is the signer
+        if (fn.params[i + 1].includes("vector")) {
+          return arg.trim().slice(1, -1).split(",");
+        } else return arg;
+      }),
```

Comment:
tested with `run` functions, and they now works.

Reply:
I found that if it's your first time running the function for your account, it will succeed.